src/Axe/wwwroot/js/site.js

```diff
@@ -1,5 +1,24 @@
 ﻿$(document).ready(function () {
 
+    //$(window).contextmenu(function (e) {
+    //    console.log(this);
+    //    //e.preventDefault();
+    //})
+
+    var pr = new Promise(function (resolve, reject) {
+        setTimeout(function () {
+            resolve('tick')
+        }, 2000);
```

Comment:
Why this timeout is so big? (2000)

Reply:
2000 ms = 2 sec isn't big. it is short but noticeable delay